test/terra/backends/qasm_simulator/qasm_noise.py

```diff
@@ -148,3 +150,26 @@ def test_kraus_gate_noise(self):
                 **self.BACKEND_OPTS).result()
             self.assertSuccess(result)
             self.compare_counts(result, [circuit], [target], delta=0.05 * shots)
+
+class QasmKrausNoiseOnQFTTests:
+    """QasmSimulator Kraus error noise model on QFT tests."""
+
+    SIMULATOR = QasmSimulator()
+    BACKEND_OPTS = {}
+
+    def test_kraus_gate_noise_on_QFT(self):
+        """Test Kraus noise on a QFT circuit"""
+
+        shots = 1000
+        noise_models = ref_kraus_noise.kraus_gate_error_noise_models_full()
+
+        for noise_model in noise_models:
+            circuit = QFT(3)
```

Comment:
It's better not to depend on the external implementation of QFT, which may change. A small circuit that manifests the bug is also more readable to the developer.

Reply:
I actually liked the idea of using QFT to reproduce the bug. I saw other tests that use QFT, so thought it was legitimate. In addition, I didn't find it easy to construct a small circuit that failed.

Reply:
ok